articles/files-response/README.md

```diff
@@ -0,0 +1,25 @@
+<p align="center">
+    <a href="https://www.angular.love/" target="_blank"><img src="https://www.angular.love/wp-content/uploads/2020/07/logo-angular.png" alt="angular love"/></a>
+</p>
+
+# Files Response
+
+## Notable libraries
+
+💖 `pg-query-stream` for streaming rows from postgres db
+
+💖 `stronger-typed-streams` for a little more safety
+
```

Comment:
we could describe typed event emitters in future

Reply:
https://valueadd.atlassian.net/browse/HON-38 😉